### PR TITLE
New TIL dealing with printing horizontally in console. (Python)

### PR DESCRIPTION
**Pull request by JoelMon**
print() can't be used to print horizontally because print() inserts a
newline to the end of the string. This TIL is a way of printing
horizontally using sys.stdout.write().

### CODE REVIEW

File: python/printing_horizontally.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Printing Horizontally
+
+The print() function can't be used to print horizontally (simulate a user 
+typing) because print() inserts a new line at the end of the string. Instead, 
+sys.stdout.write() has to be used.
+
+
+## Example
+
+    import sys
```
Comment (thorncp) [inline python/printing_horizontally.md]: Will you put this in a [fenced code block](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) and specify the language as python?

File: python/printing_horizontally.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    import time
+    
+    def slow_type(string):
+        for letter in string:
+            sys.stdout.write(letter)
+            sys.stdout.flush()
+            time.sleep(0.07) # Randon can be used to vary the speed of scroll.
```
Comment (thorncp) [inline python/printing_horizontally.md]: Typo: Randon -> Random

**Comment by JoelMon** [comment]
@thorncp, I fixed the problems with the  0adea08 commit.

**Comment by stefannibrasil** [comment]
This repo is no longer active and it's being archived. The contents will still be available as read-only. Thank you for your contributions!

**Pull request closed without merging**